content/en/docs/setup/cri.md

```diff
@@ -109,6 +109,14 @@ systemctl restart docker
 Refer to the [official Docker installation guides](https://docs.docker.com/engine/installation/)
 for more information.
 
+Please Note: 
```

Comment:
please remove the text `Please Note:` and wrap the whole block with `{{< caution >}} .... {{< /caution >}}`

example here:
https://github.com/kubernetes/website/blob/807ded0b0df21c37239e895943a8b77c6efdd0cc/content/en/docs/setup/independent/high-availability.md

also please move this block above the Docker section.

Reply:
@neolit123  Thanks for pointing it out.Corrected it. :)